cmd/testutil/json2yaml/json2yaml.go

```diff
@@ -0,0 +1,25 @@
+package main
+
+import (
+	"bytes"
+	"os"
+	"sigs.k8s.io/yaml"
+)
+
+func main() {
+	var err error
+	r := os.Stdin
+	if len(os.Args) > 1 {
+		r, err = os.Open(os.Args[1])
+		if err != nil {
+			panic(err)
+		}
+	}
+	buf := new(bytes.Buffer)
+	buf.ReadFrom(r)
+	json, err := yaml.JSONToYAML(buf.Bytes())
```

Comment:
The json variable is really the yaml payload... Might not be able to use yaml because of the package include, but it shouldn't be json.

Reply:
Thanks for catching that. Copy/paste resulted in a nonsense variable name. Fixed.